README.rst

```diff
@@ -207,6 +207,7 @@ click-odoo-makepot (stable)
                                 Only applies when --msgmerge
                                 or --msgmerge-if-new-pot are passed.
                                 [default: False]
+    --width INT                 Width of lines in the generated po files.
```

Comment:
Could you give more details here? In particular explain what happens when set to 0, and when not set (i.e. use the default behaviour of `msgmerge`).